test/relayer_test.go

```diff
@@ -26,5 +27,15 @@ func TestRelayer(t *testing.T) {
 	if testing.Short() {
 		t.Skip()
 	}
-	relayertest.TestRelayer(t, getTestChainFactory(), ibctest.NewBuiltinRelayerFactory(ibc.CosmosRly))
```

Comment:
I'm wondering the appetite for deleting this test entirely? It's duplicative. The `cmd/ibctest_test.go` seems like it should be the canonical path.

Reply:
I'll leave it in this PR. I think we may use it to ensure it's run as part of the CI test suite? As an option, we can workaround that and funnel all e2e tests through the ibctest binary.

Reply:
I'm open to removing it and using the ibctest binary in CI

Reply:
Great! I'll add ti to the list.